addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -247,7 +247,7 @@ const FontFamilyPickerUserValueWidget = SelectUserValueWidget.extend({
             googleLocalFontsEls.forEach((el, index) => {
                 $(el).append(renderToFragment('website.delete_google_font_btn', {
                     index: index,
-                    local: true,
+                    local: "true",
```

Comment:
It's probably not the way to go, having to pass true/false as string in template variable seems fragile and error prone.

What if we need the `local` variable to do more things, and somehow we have code having to pass `false` in local?
Eg, the code is doing `<i t-if="!local" `, which would be truthy even for `local: "false",`.

I am almost sure there is a way in the XML declaration to take that into account and force the true value.
I was thinking about the `="{..}"` way or something but can't make it work.
And I thought `t-attf-data-local-font="#{local}"` would work, but it does not work (it does if you add anything next to it like `t-attf-data-local-font="#{local} abc"`.

I'd really like to have confirmation from the frameworkjs that there is no workaround before moving forward with this diff.

Reply:
After discussion with @detrouxdev this should indeed be solved with OWL as the `local` property will just be a JS property read in JS when firing the delete font RPC, no more read from the DOM.

About the current diff, he suggested to use `t-att-data-local-font="local.toString()"` (like [here](https://github.com/odoo/odoo/blame/a813104ef4501fba2387d1c634cb09baa546b705/addons/web/static/src/views/fields/statusbar/statusbar_field.xml#L42) in `web`)
And to patch this line too in JS (not needed but would make the whole code "correct"):
```diff
- if (isLocalFont) {
+ if (isLocalFont === "true") {
```

I really prefer this way to go, so we don't start changing the boolean value into a string containing the boolean. 

But let's not overthink this I guess :+1:

Reply:
> 
> ```diff
> - if (isLocalFont) {
> + if (isLocalFont === "true") {
> ```

It's not rather this that you wanted to do?
> 
> ```diff
> - if (isLocalFont) {
> + if (isLocalFont !== undefined) {
> ```
I don't really have a preference for this fix... There's no ideal solution, tell me what you prefer.

Reply:
Another solution would have been to say "local" property main purpose is to show or not the attribute (and not set its value to "true" or "false"), even if also used in `t-if="local"`, and then in the JS instead of _reading_ that property (and checking its value), we could have _check_ for that attribute presence.

Eg:
```diff
- const isLocalFont = ev.target.dataset.localFont;
+ const isLocalFont = "localFont" in ev.target.dataset;
```

With this, both the XML code and JS code would be "correct" as we still pass a boolean, which show or not the attribute, and the JS is correctly in par with the XML as it checks the presence of the attribute or not.
And it would work if we were to pass `local: false` too (right now if you pass `local: "false"` it does not work because the code is not really "correct").

> There's no ideal solution

I guess this would be the ideal solution, no?

But let's go with what @qsm-odoo suggested :see_no_evil: :
>  don't think it is worth caring that much

Reply:
> I guess this would be the ideal solution, no?

It feels strange to me to add an attribute without a value, but maybe it's because I'm not used to it.

Reply:
> About the current diff, he suggested to use t-att-data-local-font="local.toString()"

Quick note: I didn't realise that the local attribute is not always defined, undefined.toString() is not ok. So my suggestion wasn't really good in this case😬